client/docs/swagger-ui/swagger.yaml

```diff
@@ -7880,2523 +7865,2188 @@ paths:
                           "value": "1.212s"
                         }
       parameters:
-        - name: denom
-          description: coin denom to query the circulating supply for
+        - name: subspace_id
+          description: Id of the subspace to query the supported reporting reasons for
           in: path
           required: true
           type: string
-        - name: divider_exponent
+          format: uint64
+        - name: pagination.key
+          description: |-
+            key is a value returned in PageResponse.next_key to begin
+            querying the next page most efficiently. Only one of offset or key
+            should be set.
+          in: query
+          required: false
+          type: string
+          format: byte
+        - name: pagination.offset
           description: >-
-            divider_exponent is a factor used to power the divider used to
-            convert the
+            offset is a numeric offset that can be used when key is unavailable.
 
-            supply to the desired representation.
+            It is less efficient than using key. Only one of offset or key
+            should
+
+            be set.
```

Comment:
Unfortunately this file is generated by the `make proto-swagger-gen` command and I don't think we should update it by hand. We can just leave it as it is since it shouldn't matter the formatting